frontend/components/tasks/dynamicAnnotation/DimensionInput.vue

```diff
@@ -0,0 +1,277 @@
+<template>
+  <div class="widget">
+    <div class="widget__answer" justify="center" align="center">
+      <div class="widget__textfield" @click="onDimensionSelectionClick">
+        <v-select
+          v-model="selected"
+          :items="dimensionOptions"
+          :label="$t('overview.dimensions')"
+          readonly
+          multiple
+          outlined
+          :required="required"
+          :rules="rules"
+        >
+          <template #selection="{ item, index }">
+            <v-chip v-if="index < 3">
+              <span>{{ item.text }}</span>
+            </v-chip>
+            <span v-if="index === 3" class="text-grey text-caption align-self-center">
+              (+{{ selected.length - 3 }} $t('overview.others'))
+            </span>
```

Comment:
Hello, there is a very minor visual bug here, I think `$t('overview.others')` should be put inside the curly bracket, otherwise it will appear as a string:
<img width="960" alt="Capture" src="https://user-images.githubusercontent.com/64476430/227880906-eeaeb491-059c-4172-af41-3419dd9382e5.PNG">

Not really important but I suppose it can be fixed quickly 😅

Reply:
lol thank you